craft_parts/executor/part_handler.py

```diff
@@ -47,6 +47,8 @@
 
 logger = logging.getLogger(__name__)
 
+_ORIGIN_PACKAGE_FORMAT = "origin_{plugin_name}_package"
```

Comment:
This is a bit strange in isolate form but I see it's following the xattr pattern already used for stage packages.